File: SuperMessenger/SuperMessenger/react-client/src/containers/Models/FileConfirmationModel.js
export default class FileConfirmationModel{
  constructor(id, previousId, sendDate, groupId, contentName ) {
    this.id = id;
    this.previousId = previousId;
    this.sendDate = sendDate;
    this.groupId = groupId;
    this.contentName = contentName;
  }
}

File: SuperMessenger/SuperMessenger/Models/FileConfirmationModel.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace SuperMessenger.Models
{
    public class FileConfirmationModel
    {
        public Guid Id { get; set; }
        public Guid PreviousId { get; set; }
        public DateTime SendDate { get; set; }
        public Guid GroupId { get; set; }
        public Guid ContentId { get; set; }
    }
}
